[2025-03-01T09:03:47.787+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-01T09:03:48.014+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.save_to_json scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-01T09:03:48.183+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.save_to_json scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-01T09:03:48.209+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-01T09:03:48.361+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_to_json> on 2025-03-01 00:00:00+00:00
[2025-03-01T09:03:48.397+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=227) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-01T09:03:48.388+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_etl', 'save_to_json', 'scheduled__2025-03-01T00:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpwnamh1d2']
[2025-03-01T09:03:48.403+0000] {standard_task_runner.py:91} INFO - Job 135: Subtask save_to_json
[2025-03-01T09:03:48.402+0000] {standard_task_runner.py:63} INFO - Started process 231 to run task
[2025-03-01T09:03:48.581+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl.save_to_json scheduled__2025-03-01T00:00:00+00:00 [running]> on host 083d33cb466d
[2025-03-01T09:03:50.516+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl' AIRFLOW_CTX_TASK_ID='save_to_json' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T00:00:00+00:00'
[2025-03-01T09:03:50.528+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-01T09:03:50.611+0000] {logging_mixin.py:188} INFO - ✅ Đã lưu 19 bản ghi vào data/2025-03/2025-03-01.jsonl
[2025-03-01T09:03:50.612+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-01T09:03:50.616+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-01T09:03:50.628+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl, task_id=save_to_json, run_id=scheduled__2025-03-01T00:00:00+00:00, execution_date=20250301T000000, start_date=20250301T090348, end_date=20250301T090350
[2025-03-01T09:03:50.705+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-01T09:03:50.730+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T09:03:50.734+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
